front/src/applications/stdcm/components/StdcmForm/StdcmConfig.tsx

```diff
@@ -173,6 +185,27 @@ const StdcmConfig = ({
     }
   }, []);
 
+  useEffect(() => {
+    if (isPathFindingLoading) {
+      setShowMessage(true);
+    }
+
+    const handleAnimationEnd = () => {
+      setShowMessage(false);
+    };
+
+    const element = document.querySelector('.pathFinding-status');
+    if (element) {
+      element.addEventListener('animationend', handleAnimationEnd);
+    }
+
+    return () => {
+      if (element) {
```

Comment:
Don't think you need to put the `if` here. With the ref, you should be able to do 
```suggestion
      pathfindingBannerRef.current?.removedEvent...
```

Reply:
Indeed, with the ref, its uselessn now. I removed it